modules/gdscript/gdscript_editor.cpp

```diff
@@ -2997,8 +2997,16 @@ Error GDScriptLanguage::complete_code(const String &p_code, const String &p_path
 							base_type.has_type = false;
 						}
 					} break;
-					case GDScriptParser::DataType::SCRIPT:
 					case GDScriptParser::DataType::GDSCRIPT: {
+						Ref<GDScript> scr = base_type.script_type;
+						if (scr.is_valid()) {
+							for (const Map<StringName, Ref<GDScript> >::Element *E = scr->get_subclasses().front(); E; E = E->next()) {
+								ScriptCodeCompletionOption option(E->key().operator String(), ScriptCodeCompletionOption::KIND_CLASS);
+								options.insert(option.display, option);
+							}
+						}
+					}
```

Comment:
Doesn't this need an explicit `[[fallthrough]]`? Not sure why it's passing CI without it.

Reply:
It's my fault. I added the macro.